crates/bevy_ecs/src/storage/blob_array.rs

```diff
@@ -256,7 +256,7 @@ impl BlobArray {
         new_capacity: NonZeroUsize,
     ) {
         #[cfg(debug_assertions)]
-        debug_assert_eq!(self.capacity, current_capacity.into());
+        debug_assert_eq!(self.capacity, usize::from(current_capacity));
```

Comment:
Question: whats the difference between this and using `get`?
```suggestion
        debug_assert_eq!(self.capacity, current_capacity.get());
```

Or maybe I’m getting the types confused?

Reply:
`get` appears to be API dedicated for this. Thanks!